go/exocom/test_helpers/index.go

```diff
@@ -0,0 +1,48 @@
+package testHelpers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/Originate/exocom/go/utils"
+)
+
+// WaitForText reads from the stream waiting for the given text
+func WaitForText(stream io.ReadCloser, text string) error {
```

Comment:
Does this have to be an `io.ReadCloser`? You seem to only need an `io.Reader` here.